src/OptionsPopup.qml

```diff
@@ -538,10 +628,34 @@ Window {
             }
         }
         if ('sshAuthorizedKeys' in settings) {
-            fieldPublicKey.text = settings.sshAuthorizedKeys
+            console.log("sshAuthorizedKeys: " +JSON.stringify(settings.sshAuthorizedKeys))
```

Comment:
Hopefully all these `console.log`s will get removed before this PR gets merged, otherwise we might upset the tinfoil-hat brigade :wink: